frontend/src/component/project/Project/ProjectFeatureToggles/ProjectFeaturesBatchActions/ArchiveButton.tsx

```diff
@@ -30,7 +29,6 @@ export const ArchiveButton: VFC<IArchiveButtonProps> = ({
     features,
     onConfirm,
 }) => {
-    const { refetch } = useProject(projectId);
```

Comment:
search refresh is passed inside onConfirm. This refetch wasn't doing anything useful